tests/static/js/testem/coverage.js

```diff
@@ -0,0 +1,18 @@
+/* globals Testem, window, XMLHttpRequest */
```

Comment:
It's a fair cop, I should have done that up front.  I added a comment with details regarding the source of the original content and its use.